lottie/src/main/java/com/airbnb/lottie/animation/keyframe/TransformKeyframeAnimation.java

```diff
@@ -172,12 +174,7 @@ public Matrix getMatrix() {
     }
 
     if (rotation != null) {
-      float rotation;
-      if (this.rotation instanceof ValueCallbackKeyframeAnimation) {
-          rotation = this.rotation.getValue();
-      } else {
-        rotation = ((FloatKeyframeAnimation) this.rotation).getFloatValue();
-      }
+      float rotation = getFloatValue(this.rotation);
```

Comment:
What about the `ValueCallbackKeyframeAnimation` case here?

Reply:
I think it can be called at following line of `Utils.getFloatValue()`.
https://github.com/airbnb/lottie-android/pull/1460/files/2f7643de191d29d1e45ba6cee478e1601e822d45#diff-cff0e0e4c6d992a385ff23491aecccc5R288